.travis.yml

```diff
@@ -1,4 +1,7 @@
 sudo: required
+arch: amd64
+os: linux
+dist: focal
```

Comment:
the first two are default but for dist the default is indeed xenial (16.04)
<img width="746" alt="image" src="https://github.com/resonai/ybt/assets/38324390/f1dc5bad-90b7-49fa-addf-63f97fa0a12f">